### PR TITLE
[lab-deep-learning] Constanze Krüger

### PR DESCRIPTION
**Pull request by Constanze05**

### CODE REVIEW

**Comment by ta-data-lis** [comment]
Hey,

To normalize the data, you should use the [normalizer](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Normalizer.html#sklearn.preprocessing.Normalizer) function. 
Try to correct that and see if the model improves.

**Comment by ta-data-lis** [comment]
Hey,

Did you check [keras optimizers](https://keras.io/optimizers/)?

Lab checked OK
